eval/mt_bench/components.py

```diff
@@ -19,6 +19,7 @@ def run_mt_bench_op(
     models_list: List[str] = None,
     models_folder: Optional[str] = None,
     device: str = None,
+    best_score_file: Optional[str] = None,
```

Comment:
Should we give this a default other than `None` like `mt-bench-best.txt` or make it required? I think this will fail at line 195 if `best_score_file` is not set.

Reply:
Mouarf, good catch!